ci/jenkins/jobs/job-templates.yaml

```diff
@@ -0,0 +1,48 @@
+- job-template:
+    name: '{name}-{test_name}-for-pull-request'
+    node: '{node}'
+    block-downstream: false
+    block-upstream: false
+    builders: '{builders}'
+    concurrent: false
+    description: '{description}'
+    project-type: freestyle
+    properties:
+    - build-discarder:
+        artifact-days-to-keep: -1
```

Comment:
Can you pass build artifact between jobs in jenkins.. Say, we want to build the image in 1 job and use that docker image is 3 other jobs